components/finder.rst

```diff
@@ -176,12 +176,12 @@ Sort the result by name or by type (directories first, then files)::
 
 You can also define your own sorting algorithm with ``sort()`` method::
 
-    $sort = function (\SplFileInfo $a, \SplFileInfo $b)
+    $sortFunction = function (\SplFileInfo $a, \SplFileInfo $b)
```

Comment:
not a big fan of this one. `$sort` describes that it is, I don't think you need the type in the name. When you have `$x = ['name 1', 'name 2']` an appropriate name for `$x` would be `$names` not `$stringNames` unless special circumstances where the type marks the difference

Reply:
I agree, what about maybe moving it directly into `->sort()` instead of assigning it to a variable?

Reply:
I would personally do it directly yeah unless there is more of such functions and ends up making it hard to read. Otherwise I think the naming (`$sort`) is perfectly correct. I'll leave @javiereguiluz make the call there :)